bears/typescript/TSLintBear.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.misc.Shell import escape_path_argument
+from coalib.results.Result import Result
+
+
+class TSLintBear(LocalBear, Lint):
+    executable = 'tslint'
+
+    def process_output(self, output, filename, file):
+        try:
+            output = json.loads("".join(output))
+        except ValueError:
+            # This implies that the result is empty and there
```

Comment:
Hey, so the basic thing is that we have a variable `output` which we are parsing json from right ?
If there is no result, can't we simply check if this is empty ?

SO, we can do something like `output == ""` or `output == ()` or `output == '{}'` or similar ?
This would allow us to do something like:

```
if output == "":
    return []
# the rest of the code
```